requirements.txt

```diff
@@ -8,7 +8,7 @@ slackeventsapi==3.0.1
 slackclient==2.9.4
 flask-cors
 asgiref==3.5.2
-lightning
+lightning==2022.10.7
```

Comment:
```suggestion
lightning==2022.10.20
```

Reply:
let's unpin this. Confirmed with platform, we should always use latest for some time.